### DISCUSSION TITLE
I was approved for GitHub Education and yet I still haven't received my benefits.

### DISCUSSION THREAD

**Discussion by manski-yuwel** [community/community#139463] [GitHub Education]
### Hi everyone,

Greetings, I would like to inquire regarding the issuance of my benefits. I was already approved and yet my benefits have yet to arrive. Can somebody help? Is there anything that can be done to speed up the process?

**Comment by davevad93** [comment]
Hi @manski-yuwel ,

Don't worry, you just have to wait. The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @manski-yuwel was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
